store/tikv/region_request.go

```diff
@@ -178,6 +188,28 @@ func (s *RegionRequestSender) sendReqToRegion(bo *Backoffer, ctx *RPCContext, re
 	return
 }
 
+func (s *RegionRequestSender) getStoreToken(st *Store, limit int64) error {
+	// Checking limit is not thread safe, preferring this for avoiding load in loop.
+	count := atomic.LoadInt64(&st.tokenCount)
+	if count < limit {
+		// Adding tokenCount is no thread safe, preferring this for avoiding check in loop.
+		atomic.AddInt64(&st.tokenCount, 1)
+		return nil
+	}
+	metrics.GetStoreLimitErrorCounter.WithLabelValues(st.addr, strconv.FormatUint(st.storeID, 10)).Inc()
+	return ErrTokenLimit.GenWithStackByArgs(st.storeID)
+
+}
+
+func (s *RegionRequestSender) releaseStoreToken(st *Store) {
+	count := atomic.LoadInt64(&st.tokenCount)
+	// Decreasing tokenCount is no thread safe, preferring this for avoiding check in loop.
+	if count > 0 {
+		atomic.AddInt64(&st.tokenCount, -1)
+	}
+	logutil.BgLogger().Warn("release store token failed, count equals to 0")
```

Comment:
Seems it's an unexpected log if `count > 0`.

Reply:
Yeah, just a log in case of some bug.

Reply:
But it always logs because there is no return at L207.

Reply:
my bad，done